bitnami/postgresql/templates/primary/statefulset.yaml

```diff
@@ -534,19 +534,18 @@ spec:
           env:
             - name: DATA_SOURCE_URI
               value: {{ printf "127.0.0.1:%d/postgres?sslmode=disable" (int (include "postgresql.v1.service.port" .)) }}
-            {{- $pwdKey := ternary (include "postgresql.v1.adminPasswordKey" .) (include "postgresql.v1.userPasswordKey" .) (or (eq $customUser "postgres") (empty $customUser)) }}
             {{- if .Values.auth.usePasswordFiles }}
             - name: DATA_SOURCE_PASS_FILE
-              value: {{ printf "/opt/bitnami/postgresql/secrets/%s" $pwdKey }}
+              value: {{ printf "/opt/bitnami/postgresql/secrets/%s" (include "postgresql.v1.adminPasswordKey" .) }}
```

Comment:
Hello, even though I have zero knowledge on all of this, are these changes working?

If we look at the following configuration, `adminPasswordKey` and `adminPasswordKey` is only used if  `existingSecret` is set, then `auth.*` field would be ignored.
https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml#L162

Should these `metrics` configuration considere both, `auth.*` and/or `existingSecret` fields? I'm asking because I copied these changes and I still have this error `ERROR:  permission denied for function pg_ls_waldi`